docs/faq.rst

```diff
@@ -265,6 +265,27 @@ In your Conda environment file (``environment.yml``)::
     -e ..
 
 
+Can I use Anaconda Project and ``anaconda-project.yml``
+-------------------------------------------------------
+
+Yes. With ``anaconda-project>=0.8.4`` you can use the `Anaconda Project`_ configuration
+files ``anaconda-project.yaml`` (or ``anaconda-project.yml``) directly in place of a
+Conda environment file by using ``dependencies:`` as an alias for ``packages:``.
+
+I.e., your ``anaconda-project.yaml`` file can be used as an ``environment.yml`` file if
+it contains::
+
+    dependencies:
+      - python=3.9
+      - scipy
+      ...
+
+For an example, see `this project
+<https://physics-581-physics-inspired-computational-techniques.readthedocs.io/en/latest/>`_.
```

Comment:
I don't think it's needed to mention a project that is already using this. I'd say that the correct docs for it should be enough.